index.html

```diff
@@ -10,25 +10,148 @@
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <div class="box1">
+        <img src="/image/Snapshoot Portfolio (3).png">
+        <h1>Multi-Post<br>Stories
+        <ul>
+            <li class="point1">CANOPY</li>
+            <li class="point2">Back End Dev</li>
+            <li class="point3">2015</li>
+        </ul>
+    </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <!-- About me Section -->
+    <div class="about">
+        <div class="base">
+            <div class="top">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social">
+            <a href="#">LET'S CONNECT</a>
+            <ul class="icons">
                 <li><img src="/image/Vector.svg"></li>
                 <li><img src="/image/Linkedin icon.svg"></li>
                 <li><img src="/image/Vector (1).svg"></li>
                 <li><img src="/image/Vector (2).svg"></li>
                 <li><img src="/image/Vector (3).svg"></li>
-
             </ul>
-    </section>
-</body>
+        </div>
+        <div class="btn">
+            <a href="#" class="resume">Get my resume</a>
+        </div>
+        <div class="skill_list">
+            <div class="title1">
+                <h1>Languages</h1>
+                <a href="#" class= "fa fa-angle-down"></a>
+                <div class="langlist">
+                <ul class="list1">
+                    <img src="/image/Ellipse 1.png">
+                    <p>JavaScript</p>
+                </ul>
+                <ul class="list2">
+                    <img src="/image/Ellipse 1 (1).png" alt="">
+                    <p>HTML</p>
+                </ul>
+                <ul class="list3">
+                    <img src="/image/Ellipse 1 (2).png">
+                    <p>CSS</p>
+                </ul>
+                </div>
+            </div>
+            <div class="title2">
+                <h1>Frameworks</h1>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+            <div class="title3">
+                <h1>Skills</h1>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+        </div>
+        </div>
+    </div>
```

Comment:
- [ ] Overall, this is what you have:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/243097474-b82270f3-8a94-4af2-8194-37bdb1e61879.png)|![image](https://user-images.githubusercontent.com/99715640/243097525-3bd58539-b9e1-49a3-b83f-cd508ab8a055.png)|

  - [ ] Your images are not showing which might be due to choice of filenames for the image tags. You have spaces between and you did not escape those spaces. I would recommend renaming the image files without space in them. E.g. multistore.png or multi_store.png.
  - [ ] Your buttons are missing important styles for all their states - active, hover, pressed. Kindly use the palette provided in the Figma design to implement this feature. Here's what it looks like:
![image](https://github.com/Nomarg/My-Portfolio/assets/99715640/d9d6e86e-c112-4226-a7b3-9150976329fb)

  - [ ] An import section from previous milestones is missing - the header and the landing page.
  - [ ] Each project cards seem longer than necessary. Kindly check the Figma design and ensure to stick to it as close as you can.
  - [ ] Kindly compare the designs and implement the necessary changes - I really hope you find my suggestions helpful because I have much believe in your ability. :+1: